### PR TITLE
Tailwind breakpoints

### LINKED ISSUE #391: Use Tailwind default screen breakpoints and/or add one for smaller screens
## Description

Currently our configuration for screens sizes in Tailwind looks like this:

```js
screens: {
  // Determined by the lower bound of screen width
  tab: '768px',
  desk: '1024px',
  wide: '1216px',
  hd: '1408px',
},
```
It uses ambiguous custom names, since `desk` could span all screens that are larger than `tab` and `wide` and `hd` feels unintuitive.

We should consider renaming those to more generic and standard names as [Tailwind defaults](https://tailwindcss.com/docs/breakpoints) (`md`, `lg`, `xl` ...), and include a breakpoint for smaller devices; frequent options are `480px` and `640px`.

## Alternatives

Make changes specific to smaller screens manual with media queries on each page. I don't think we really want to go this path.

## Additional context

The mockups for [mobile designs in Figma](https://www.figma.com/file/w60dl1XPUvSaRncv1Utmnb/Openverse-Releases?node-id=133%3A11555) have a width of **375px**. 
@panchovm can also help by sharing what he uses here. 

## Implementation

- [x] 🙋 I would be interested in implementing this feature once we agree on using the default tailwind names or new ones :)

### PR DESCRIPTION
**Pull request by krysal**
## Fixes

Fixes #391 by @krysal 

## Description

This PR removes the custom screen breakpoints in favor of Tailwind defaults. It updates classes prefixed with old names, for example `desk:hidden` to `lg:hidden` (min-width: 1024px) and `tab:px-0` to `md:px-0` (min-width: 768px).

Also replaces the Bulma's named mixin `@include desktop` with the Tailwind equivalent `@screen lg`. This is not possible to do with mixins `@include tablet` and `@include mobile` as they use `max-width` instead of `min-width`, so no equivalent exists yet.

## Testing Instructions

Run the project and see everything remains with the same styles, also all unit tests pass.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x]  I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/pages/search.vue [hunk 1]
```diff
@@ -150,5 +150,22 @@ export default BrowsePage
...
+  @include mobile {
+    width: 100%;
+    flex: none;
+  }
+}
+.grid-sidebar {
+  padding: 0;
+  border-right: 1px solid $color-transition-gray;
+  width: 21.875rem;
+}
```
Comment (dhruvkb) [inline src/pages/search.vue:170]: Can we change these to TW utils and drop the SCSS entirely?

**Review by dhruvkb — APPROVED** [review]
The changes look good to me! Reduction in custom configuration is always a good thing.

File: src/pages/search.vue (hunk 1 above)
Comment (obulat) [reply src/pages/search.vue:170]: This page will need to change soon, so I would postpone it to another PR, @dhruvkb

**Review by obulat — APPROVED** [review]
Thank you for synchronizing our breakpoints with the standard ones, using them puts less cognitive load on us.

File: src/pages/search.vue (hunk 1 above)
Comment (krysal) [reply src/pages/search.vue:170]: I thought the same as you, the redesign should definitely reduce the custom css to what is not included with Tailwind.

**Pull request merged**